layouts/partials/head.html

```diff
@@ -10,8 +10,12 @@
   gtag('config', 'UA-36037335-10');
 </script>
 
-<!-- Docsy head.html begins here -->
+<!-- alternative translations -->
+{{ range .Translations -}}
+<link rel="alternate" hreflang="{{ .Language.Lang }}" href="{{ .Permalink }}">
+{{ end -}}
```

Comment:
```suggestion
{{ end -}}
<link rel="alternate" hreflang="x-default" href="{{ .Permalink }}">
```

Google recommends to add `hreflang="x-default"` as a fallback. (ref. [Tell Google about localized versions of your page - Search Console Help](https://support.google.com/webmasters/answer/189077?hl=en)) This allows visitors to view the English page first in case the page has not localized yet.

Reply:
No, `{{ .Permalink }}` is not replaced with an English page when viewing a localized one. This should be changed to something like `.Translations[en].Permalink || .Permalink`.

Reply:
```suggestion
{{ end -}}
{{ with index (where .AllTranslations "Language.Lang" "en") 0 -}}
<link rel="alternate" hreflang="x-default" href="{{ .Permalink }}">
{{ end -}}
```

This one should always generate `x-default` of  English page:

**For English page**
```html
<link rel="alternate" hreflang="zh" href="http://localhost:1313/zh/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="ko" href="http://localhost:1313/ko/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="ja" href="http://localhost:1313/ja/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="fr" href="http://localhost:1313/fr/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="id" href="http://localhost:1313/id/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="x-default" href="http://localhost:1313/docs/concepts/workloads/pods/pod-overview/">
```

**For non-English page**
```html
<link rel="alternate" hreflang="en" href="http://localhost:1313/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="zh" href="http://localhost:1313/zh/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="ko" href="http://localhost:1313/ko/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="fr" href="http://localhost:1313/fr/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="id" href="http://localhost:1313/id/docs/concepts/workloads/pods/pod-overview/">
<link rel="alternate" hreflang="x-default" href="http://localhost:1313/docs/concepts/workloads/pods/pod-overview/">
```

Reply:
I'd prefer to push the general change first and then consider a separate PR that caters to Google's specific whims (ie, the nonstandard `x-default` language).

Reply:
OK, it's reasonable. Please skip this change for now.